### PR TITLE
Add Sysdig to Remote Endpoints and Storage

### PR DESCRIPTION
**Pull request by dmitsh**
Signed-off-by: Dmitry Shmulevich <dmitry.shmulevich@sysdig.com>

### CODE REVIEW

File: content/docs/operating/integrations.md [hunk 1]
```diff
@@ -55,6 +55,7 @@ data volumes.
   * [QuasarDB](https://doc.quasardb.net/master/user-guide/integration/prometheus.html): read and write
   * [SignalFx](https://github.com/signalfx/metricproxy#prometheus): write
   * [Splunk](https://github.com/kebe7jun/ropee): read and write
+  * [Sysdig](https://sysdig.com/opensource/prometheus): read
```
Comment (brian-brazil) [inline content/docs/operating/integrations.md:58]: To be clear a normal Prometheus can read data from Sysdig via the remote read protocol?

We've not had one of those before for this list, and I'm not seeing any relevant docs on the linked page so I'm just double checking.

Comment (dmitsh) [reply content/docs/operating/integrations.md]: Sysdig does not support Prometheus Remote Read API, but it exposes Prometheus HTTP API endpoint for querying the data.

Comment (dmitsh) [reply content/docs/operating/integrations.md]: I've also updated the link

Comment (brian-brazil) [reply content/docs/operating/integrations.md]: This list is for remote storage integrations, we don't have a list for things that provide the HTTP API I'm afraid.

**Comment by dmitsh** [comment]
Got it, thanks!

**Pull request closed without merging**